extensions/amp-bind/0.1/bind-impl.js

```diff
@@ -170,12 +186,76 @@ export class Bind {
       dev().fine(TAG, `Initialized ${bindings.length} bindings from ` +
           `${boundElements.length} elements.`);
     });
+    return this.scanPromise_;
+  }
+
+  /**
+   * Scans the substree rooted at `rootElement` and removes bindings for nodes
+   * that contain bindable elements. This function is not idempotent. To add
+   * bindings for a subtree, see #addBindingsForSubtree. Returns a promise
+   * that resolves after bindings have been removed.
+   * @param {!Element} rootElement
+   * @return {Promise}
+   */
+  removeBindingsForSubtree(rootElement) {
+    this.scanPromise_ = this.scanSubtree_(rootElement).then(results => {
```

Comment:
I don't think we need to scan the subtree here since we should already have the bindings/expression strings that we're about to remove.